Author: Aditya Vaidya <kroq.gar78@gmail.com>
Forwarded: no
Description: Remove the mimetype registration in-game because the package does it
diff --git a/src/main.c b/src/main.c
index d61e015..64b2b89 100644
--- a/src/main.c
+++ b/src/main.c
@@ -1297,7 +1297,7 @@ int main(int argc, char *argv[])
 					break;
 				}
 			}
-			if (sdl_key=='i' && (sdl_mod & KMOD_CTRL))
+			/*if (sdl_key=='i' && (sdl_mod & KMOD_CTRL))
 			{
 				if(confirm_ui(vid_buf, "Install Powder Toy", "You are about to install The Powder Toy", "Install"))
 				{
@@ -1310,7 +1310,7 @@ int main(int argc, char *argv[])
 						error_ui(vid_buf, 0, "Install failed - You may not have permission or you may be on a platform that does not support installation");
 					}
 				}
-			}
+			}*/
 			if (sdl_key=='f')
 			{
 				framerender = 1;
